### DISCUSSION TITLE
Why is the imageUrl absolute instead of relative ? (userInfo table)

### DISCUSSION THREAD

**Discussion by dufguix** [serverpod/serverpod#2184] [Q&A]
Hi

Is there a reason to have a full path for the image url?

The url stored in the db starts with 'http://localhost:8080/'.

In production, if we want to change the domain name, it will not be seamless.
I suggest to build the path based on an environment var.

**Comment by Isakdl** [comment]
I agree with you on this, issue on this topic: https://github.com/serverpod/serverpod/issues/904

**Comment by dufguix** [comment]
ok thx
